code/index.html

```diff
@@ -1,30 +1,248 @@
+        <h2>NEW PRODUCTS</h2>
+        <div class="accordion-section__display"><span class="accordion-section__display__button"></span>
+          <div class="accordion__image"><img src="./images/red.png" alt="red bottle"></div>
+          <p>Product 1</p>
+        </div>
+        <div class="accordion-section__expand">
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Alias necessitatibus ipsam maxime possimus
+            voluptatibus,
+            et laborum expedita atque officia ullam voluptatem consequuntur omnis cum dolor est! 
+            Pariatur facere aspernatur mollitia!</p>
+        </div>
+        <div class="accordion-section__display"><span class="accordion-section__display__button"></span>
+          <div class="accordion__image"><img src="./images/blue.png" alt="blue bottle"></div>
+          <p>Product 2</p>
+        </div>
+        <div class="accordion-section__expand">
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Alias necessitatibus ipsam maxime possimus
+            voluptatibus,
+            et laborum expedita atque officia ullam voluptatem consequuntur omnis cum dolor est! 
+            Pariatur facere aspernatur mollitia!</p>
+        </div>
+        <div class="accordion-section__display"><span class="accordion-section__display__button"></span>
+          <div class="accordion__image"><img src="./images/green.png" alt="green bottle"></div>
+          <p>Product 3</p>
+        </div>
+        <div class="accordion-section__expand">
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Alias necessitatibus ipsam maxime possimus
+            voluptatibus,
+            et laborum expedita atque officia ullam voluptatem consequuntur omnis cum dolor est! 
+            Pariatur facere aspernatur mollitia!</p>
+        </div>
+        <div class="accordion-section__display"><span class="accordion-section__display__button"></span>
+          <div class="accordion__image"><img src="./images/purple.png" alt="purple bottle"></div>
+          <p>Product 4</p>
+        </div>
+        <div class="accordion-section__expand">
+          <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Alias necessitatibus ipsam maxime possimus
+            voluptatibus,
+            et laborum expedita atque officia ullam voluptatem consequuntur omnis cum dolor est! 
+            Pariatur facere aspernatur mollitia!</p>
+        </div>
       </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+  </main>
+
+  <!--back to top button-->
+  <section class="top">
+    <a href="#top" class="top-button">^</a></section>
+
+  <footer class="footer">
+    <div class="footer-wrapper">
+      <h1 class="footer-wrapper__logo"> N|C
+      </h1>
+      <p>&copy; Pauline Andersson 2021</p>
+      <div class="footer-wrapper__social">
+        <div class="footer-wrapper__social__icon"><img src="./images/Fb.svg"></div>
```

Comment:
Indentations over here as well.
Moreover, both file names and directory names should be lowercase, so instead of `Fb.svg` we should name it `fb.svg`